src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -567,6 +570,30 @@ void platform_pm_runtime_enable(uint32_t context, uint32_t index)
 	}
 }
 
+void platform_pm_runtime_waiti_writeback_en(uint32_t index)
+{
+	struct pm_runtime_data *prd = pm_runtime_data_get();
+	struct cavs_pm_runtime_data *pprd = prd->platform_data;
+
+	pprd->wait_writeback_core_mask |= BIT(index);
```

Comment:
Maybe I've mentioned this in another patch, but dont we need to lock this mask ?

Reply:
I think we should